process_request/routines.py

```diff
@@ -34,58 +34,37 @@ def get_data(self, item):
         obj = aspace.client.get(item, params={"resolve": ["resource::linked_agents", "ancestors", "top_container", "top_container::container_locations"]})
         if obj.status_code == 200:
             as_data = {}
-            item_collection = obj.json().get("ancestors")[-1].get("_resolved")
-            as_data['creator'] = get_collection_creator(obj.json())
+            item_json = obj.json()
+            item_collection = item_json.get("ancestors")[-1].get("_resolved")
+            as_data['creator'] = get_collection_creator(item_json)
             as_data['restrictions'] = "TK"
+            as_data['restrictions_text'] = "TK"
             as_data['collection_name'] = item_collection.get("title")
-            if len(obj.json().get("ancestors")) > 1:
-                as_data['aggregation'] = obj.json().get("ancestors")[0].get("_resolved").get("display_string")
+            if len(item_json.get("ancestors")) > 1:
```

Comment:
I could be wrong about this, but I believe that an empty list is falsy, so you should be able to just do 
```suggestion
            if item_json.get("ancestors"):
```

Reply:
A collection is an ancestor, which is why I was checking that there's more than 1 ancestor.